packages/backend-modules/republik/migrations/sqls/20250218154508-user-birthyear-up.sql

```diff
@@ -0,0 +1,3 @@
+-- migrate up here: CREATE TABLE...
+ALTER TABLE "public"."users" ADD COLUMN "birthyear" integer;
+
```

Comment:
I would add the proposed birth year column migration as part of the up migration.

We did the same for the [profile link migration](https://github.com/republik/plattform/blob/27ae21bff3308552d14240dad07d4d646ca1da43/packages/backend-modules/auth/migrations/sqls/20241212154336-profileLink-up.sql).